acceptance-tests/src/test/resources/features/site/publication.feature

```diff
@@ -100,3 +100,16 @@ Feature: Ensure publication page displays required fields.
             | 1.9 Dataset  |
             | 1.10 Dataset |
             | 2.0 Dataset  |
+
+    Scenario: National Statistics kite mark on National statistic information type only
+        Given I navigate to the "search" page
+        When I search for ""
+        Then I click on the "National statistics" link
+        And I can click on the "Morbi tempor euismod vehicula" link
```

Comment:
Do you check for the icon on the search page or just the publication page?
Probably worth checking both.

This would probably read better as Given -> When -> And -> Then ...

Reply:
Will add a separate check on the search page too.